src/test/lib/storage/index/partial_hash/partial_hash_index_test.cpp

```diff
@@ -0,0 +1,755 @@
+#include <memory>
+#include <set>
+#include <unordered_set>
+#include <vector>
+
+#include "base_test.hpp"
+
+#include "storage/index/partial_hash/partial_hash_index.hpp"
+#include "types.hpp"
+
+namespace hyrise {
+
+class PartialHashIndexTest : public BaseTest {
+ protected:
+  void SetUp() override {
+    TableColumnDefinitions table_column_definitions;
```

Comment:
picky: auto to stick? If you only emplace one element, I think the constructor w/ initializer list is also quite readable.